studio/components/grid/components/header/RLSBannerWarning.tsx

```diff
@@ -41,7 +41,7 @@ export default function RLSBannerWarning() {
             <div className="ml-20 absolute right-2">
               <Button
                 type="outline"
-                className="text-white hover:text-scale-1200"
```

Comment:
this might be wrong on dark mode

Reply:
@Hallidayo would you mind taking a screenshot of the dark mode too?